model/requirement-upload.js

```diff
@@ -155,6 +155,7 @@ module.exports = memoize(function (db) {
 				statusLog.push({
 					label: log.getOwnDescriptor('label').valueToJSON(),
 					time: log.getOwnDescriptor('time').valueToJSON(),
+					officialFullName: log.getOwnDescriptor('officialFullName').valueToJSON(),
```

Comment:
We do not store `officialFullName` in status logs for uploads, it's only certificates that have them at this point, still as I see we do not include statusLog for certs snapshot (that's probably bug, I've opened issue for that at #1754)